deploy/docs/Best_Practices.md

```diff
@@ -1030,6 +1031,90 @@ metadata:
         value: '12345'
 ```
 
+## Using newer Kube Prometheus Stack
+
+Due to breaking changes, we do not support the latest Kube Prometheus Stack.
+We are aware that it can be a major issue, so this section describes how to install Kube Prometheus Stack in version `34.10.0`
+to work with our collection.
+
+1. Prepare [values-prometheus.yaml] with Kube Prometheus Stack configuration:
```

Comment:
Maybe it worthy to mention that for default configuration customer can use just prepared `values-prometheus.yaml`

Reply:
Not sure about it, as this can be quickly out of date

Reply:
ok, you are right, we are not ideal and we will forget about updating this :(